### PR TITLE
Translate Multiple vulnerabilities in Ruby (pt)

### PR DESCRIPTION
**Pull request by fpgentil**
@lucianosousa @alanwillms @jcserracampos

### CODE REVIEW

File: pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+É altamente recomendado para os usuários do Ruby que sigam uma das soluções alternativas o mais
+rápido possível.
+
+## Versões Afetadas
+
+* Série do Ruby 2.2: 2.2.7 and anteriores
+* Série do Ruby 2.3: 2.3.4 and anteriores
+* Série do Ruby 2.4: 2.4.1 and anteriores
+* anteriores ao _trunk revision_ 59672
```
Comment (alanwillms) [inline pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: ficou `and anteriores` ao invés de `e anteriores`

File: pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Soluções alternativas
+
+Até o momento, não existem _releases_ do Ruby incluindo correções para o RubyGems.
+Mas você pode atualizar o RubyGems para a última versão.
+RubyGems 2.6.13 ou versões superiores à correção das vulnerabilidades.
```
Comment (alanwillms) [inline pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: Faltou a palavra `includes`, ficaria algo como: `RubyGems 2.6.13 ou versões mais recentes incluem a correção das vulnerabilidades.`

File: pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+Se você não pode atualizar o RubyGems, você pode aplicar os seguintes _patches_ como soluções
+alternativas.
+
+* [para o Ruby 2.2.7](https://bugs.ruby-lang.org/attachments/download/6690/rubygems-2613-ruby22.patch)
+* [para o Ruby 2.3.4](https://bugs.ruby-lang.org/attachments/download/6691/rubygems-2613-ruby23.patch)
+* para o Ruby 2.4.1: são necessários 2 _patches_.  Aplique-os sequencialmente:
+  1. [RubyGems 2.6.11 até 2.6.12](https://bugs.ruby-lang.org/attachments/download/6692/rubygems-2612-ruby24.patch)
+  2. [RubyGems 2.6.12 até 2.6.13](https://bugs.ruby-lang.org/attachments/download/6693/rubygems-2613-ruby24.patch)
+
+Sobre o _trunk_, atualize para a última versão.
```
Comment (alanwillms) [inline pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: `Quanto ao _trunk_` talvez soe melhor, o que acha?

File: pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Creditos
```
Comment (alanwillms) [inline pt/news/_posts/2017-08-29-multiple-vulnerabilities-in-rubygems.md]: `Créditos`

**Comment by alanwillms** [comment]
I've added a few minor suggestions. Thanks!

**Comment by fpgentil** [comment]
I've updated the PR, thanks for the review! 😄

**Comment by hsbt** [comment]
@fpgentil Thanks for your translation. Can you add CVE numbers to vulnerability list? see https://github.com/ruby/www.ruby-lang.org/pull/1628/files

**Comment by hsbt** [comment]
and https://github.com/ruby/www.ruby-lang.org/commit/1ad743efe4957280c125d8183ff9e87c8a45c611

**Comment by fpgentil** [comment]
Done, @hsbt 😄

**Comment by hsbt** [comment]
Thanks a lot! I will release your translations.

**Pull request merged**